### PR TITLE
Awesome book javascript pull Request

### PR DESCRIPTION
**Pull request by Atril33**
**Awesome Books**
This is Awesome Books Project is created with _HTML, CSS & JS_.
This Project has two `<Inputs>` field and one Add ` <button>`.
We Used `localStorage` Function in this Project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,254 @@
...
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
```
Comment (mahbubul14) [inline README.md]: - [ ] Kindly update the getting started section. This should be an example of how you may give instructions on setting up your project locally. :+1:

File: README.md [hunk 2]
```diff
@@ -1 +1,254 @@
...
+- HTML
+- CSS
+- JAVASCRIPT
+# <a name="authors">Syed Atril & Peter</a>
+👤 **Syed Atril**
+
+- GitHub: [@githubhandle](https://github.com/Atril33)
+- Twitter: [@twitterhandle](https://twitter.com/AtrilSyed)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/syed-atril-831696248/)
+- Facebook [Facebook](https://web.facebook.com/profile.php?id=100073724910623)
```
Comment (mahbubul14) [inline README.md:210]: - [ ] Kindly add another author's info on this project as well. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Atril33 & @peter-akhigbe,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Munsa1 — APPROVED** [review]
Hi @Atril33 @peter-akhigbe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made requested changes by previous reviewer :heavy_check_mark: 
 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**